### PR TITLE
feat: add mainnet timestamp for shanghai

### PR DESCRIPTION
**Pull request by onbjerg**
Ref https://github.com/ethereum/execution-specs/pull/727

### CODE REVIEW

File: crates/primitives/src/chain/spec.rs [hunk 1]
```diff
@@ -778,133 +789,6 @@ mod tests {
         );
     }
 
-    /// Checks that time-based forks work
-    ///
-    /// This is based off of the test vectors here: https://github.com/ethereum/go-ethereum/blob/5c8cc10d1e05c23ff1108022f4150749e73c0ca1/core/forkid/forkid_test.go#L155-L188
-    #[test]
-    fn timestamped_forks() {
```
Comment (rkrasiuk) [inline crates/primitives/src/chain/spec.rs:785]: @onbjerg what's the reason behind removing this test?

**Review by rkrasiuk — APPROVED** [review]
lgtm modulo test removal

File: crates/primitives/src/chain/spec.rs (hunk 1 above)
Comment (onbjerg) [reply crates/primitives/src/chain/spec.rs:785]: Described in 41053eb:

This was a temporary test used to check that timestamped
forks work, but now we can test it using the actual chain
specs for e.g. mainnet instead.

Comment (onbjerg) [reply crates/primitives/src/chain/spec.rs]: Re-added the test to unblock

**Review by mattsse — APPROVED** [review]

**Pull request merged**